src/test/scala/it/pagopa/interop/purposeprocess/SpecData.scala

```diff
@@ -74,6 +75,20 @@ object SpecData {
     name = "test_name"
   )
 
+  val riskAnalysisServiceNoVersion: Map[TenantKind, Map[String, RiskAnalysisFormConfig]] =
+    Map(TenantKind.PA -> Map.empty, TenantKind.PRIVATE -> Map.empty, TenantKind.GSP -> Map.empty)
+
+  val riskAnalysisServiceComplete: Map[TenantKind, Map[String, RiskAnalysisFormConfig]] = Map(
+    TenantKind.PA      -> Map("1.0" -> RiskAnalysisFormConfig(version = "1.0", Nil)),
+    TenantKind.PRIVATE -> Map("1.0" -> RiskAnalysisFormConfig(version = "1.0", Nil)),
+    TenantKind.GSP     -> Map("1.0" -> RiskAnalysisFormConfig(version = "1.0", Nil))
+  )
+
+  val riskAnalysisServiceNoPA: Map[TenantKind, Map[String, RiskAnalysisFormConfig]] = Map(
+    TenantKind.PRIVATE -> Map("1.0" -> RiskAnalysisFormConfig(version = "1.0", Nil)),
+    TenantKind.GSP     -> Map("1.0" -> RiskAnalysisFormConfig(version = "1.0", Nil))
+  )
+
```

Comment:
No need to mock these, testing the real configurations is a more valuable test in my opinion
```suggestion
```